RussianFilter/sections/specific.txt

```diff
@@ -20,6 +20,8 @@ Ly9hbi55YW5kZXgucnUv
 !
 tut.by##.l-main > div[class^="b-top b-"] > div[class^="news-section-"]
 /images/bnn.gif$domain=nebesa.mobi
+||nudecelebs.pro/img/bg/*.gif
+nudecelebs.pro##.custom_block
```

Comment:
Too generic name, better include parent elements.

Reply:
I removed one obsolete rule and exchanged simple rule to ext-css.